src/dom/context2d.js

```diff
@@ -1,22 +1,20 @@
 export default function(width, height, options) {
-  var colorSpace = "srgb";
   var scale = devicePixelRatio;
-  if (options == +options) {
+  if (options == null) {
+    options = {};
+  } else if (options == +options) {
```

Comment:
This should be strict based on the type (`typeof options === "number"`) rather than using coercion.

Reply:
Trying to be fully compatible with the current implementation, I wanted to support dpr passed as a string. Not sure why anyone would do that, though :)

Reply:
Full compatibility is not practical; someone could pass an object that implements valueOf, and this would have worked with the old API, but is ambiguous with named options under the new API. We should only support the intended pattern of passing a number.